src/pages/Home/hooks/useGetTestimonials.js

```diff
@@ -0,0 +1,120 @@
+import axios from "axios";
+import { useEffect, useState } from "react";
+
+
+export default function useGetTestimonials() {
+    const [testimonials, setTestimonials] = useState([
```

Comment:
All constants, like these and others, should be moved to `srs/constants` file in imported here, not placed in the page itself.